CONTRIBUTING.md

```diff
@@ -2,247 +2,41 @@
-   import sklearn; print("Scikit-Learn", sklearn.__version__)
-   ```
-
--  Please be specific about what estimators and/or functions are involved
-   and the shape of the data, as appropriate; please include a
-   [reproducible](https://stackoverflow.com/help/mcve) code snippet
-   or link to a [gist](https://gist.github.com). If an exception is raised,
-   please provide the traceback.
-
-New contributor tips
---------------------
-
-A great way to start contributing to scikit-learn is to pick an item from the
-list of
-[good first issues](https://github.com/scikit-learn/scikit-learn/labels/good%20first%20issue). If
-you have already contributed to scikit-learn look at
-[Easy issues](https://github.com/scikit-learn/scikit-learn/labels/Easy)
-instead. Resolving these issues allow you to start contributing to the project
-without much prior knowledge. Your assistance in this area will be greatly
-appreciated by the more experienced developers as it helps free up their time to
-concentrate on other issues.
-
-Documentation
--------------
-
-We are glad to accept any sort of documentation: function docstrings,
-reStructuredText documents (like this one), tutorials, etc.
-reStructuredText documents live in the source code repository under the
-doc/ directory.
-
-You can edit the documentation using any text editor and then generate
-the HTML output by typing ``make html`` from the doc/ directory.
-Alternatively, ``make`` can be used to quickly generate the
-documentation without the example gallery. The resulting HTML files will
-be placed in ``_build/html/stable`` and are viewable in a web browser. See the
-``README`` file in the ``doc/`` directory for more information.
-
-For building the documentation, you will need
-[sphinx](http://sphinx.pocoo.org/),
-[matplotlib](https://matplotlib.org/), and
-[pillow](https://pillow.readthedocs.io/en/latest/).
-
-When you are writing documentation, it is important to keep a good
-compromise between mathematical and algorithmic details, and give
-intuition to the reader on what the algorithm does. It is best to always
-start with a small paragraph with a hand-waving explanation of what the
-method does to the data and a figure (coming from an example)
-illustrating it.
-
-Further Information
--------------------
-
-Visit the [Contributing Code](http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines)
-section of the website for more information including conforming to the
-API spec and profiling contributed code.
+There are many ways to contribute to scikit-learn, with the most common ones
+being contribution of code or documentation to the project. Improving the
+documentation is no less important than improving the library itself. If you
+find a typo in the documentation, or have made improvements, do not hesitate to
+send an email to the mailing list or preferably submit a GitHub pull request.
+Full documentation can be found under the
```

Comment:
It's not honest to say "Full documentation" as the documentation includes API docstrings and the example gallery, which do not live in `doc`. Perhaps just say "Documentation"

Reply:
done ✅